docs/imagecustomizer/api/cosi.md

```diff
@@ -45,6 +45,12 @@ compression.
 All partition image files MUST be in the `images` directory or one of its
 subdirectories.
 
+#### Compression Options
```

Comment:
The compression level doesn't affect the file format. So, it probably doesn't need to be mentioned in this file.